packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -490,6 +590,7 @@ export default class DropdownCore extends React.Component<
                     dropdownStyle,
                 ]}
             >
+                {this.maybeRenderSearchTextInput()}
                 {items.map((item, index) => {
```

Comment:
I think this map function should be a refactored into a member call and we could use reduce to build the focusable count. No change requested. Just a consideration for a future refactor.

Reply:
Good call! I might do this refactor in a new PR that integrates `react-window` into the `Dropdown` component.